src/components/apis/list-of-apis/ko/runtime/api-list.html

```diff
@@ -127,28 +129,68 @@
     <!-- ko if: hasPager -->
```

Comment:
`<!-- ko if: hasPager -->` and `data-bind="visible: needPaging"` seem like solving the same problem. Probably having `<!-- ko if: needPaging -->` on top level would be enough.